WcaOnRails/app/jobs/update_wca_registration.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+#
+require_relative '../helpers/microservices_helper'
+class UpdateWCARegistration < ApplicationJob
+  def perform(status)
+    token = get_wca_token
```

Comment:
If you only ever use this once, you can just as well inline it

Reply:
we will definitely use get_wca_token again for communicating with user microservices!

Reply:
My bad, the comment sounds ambiguous. What I meant is: If you only use **this variable** once, you might as well inline it.